package.json

```diff
@@ -42,12 +42,48 @@
 					"title": "Neovim path",
 					"description": "Full path to Neovim executable. If you have enabled useWSL specify the linux path to nvim executable"
 				},
+				"vscode-neovim.neovimOperatingSystemPaths.windows": {
+					"type": "string",
+					"default": "",
+					"title": "Neovim path on Windows",
+					"description": "Full path to Neovim executable that should be used if using Code on Windows. If the neovimPath setting is not empty exists it will be used instead of this setting"
+				},
+				"vscode-neovim.neovimOperatingSystemPaths.osx": {
+					"type": "string",
+					"default": "",
+					"title": "Neovim path on OSX",
+					"description": "Full path to Neovim executable that should be used if using Code on OSX. If the neovimPath setting is not empty it will be used instead of this setting"
+				},
+				"vscode-neovim.neovimOperatingSystemPaths.linux": {
```

Comment:
It should just be `neovimPath.windows` instead of the super lenghty name. Dunno why I changed it